dss/events/handlers/index.py

```diff
@@ -79,7 +92,7 @@ def __init__(self, replica: Replica, bundle_id: str, logger: logging.Logger) ->
         self.bundle_id = bundle_id
 
     @classmethod
-    def from_bucket(cls, replica: Replica, bucket_name: str, key: str, logger):  # TODO: return type hint
+    def from_bucket(cls, bucket_name: str, key: str, replica: Replica, logger):  # TODO: return type hint
```

Comment:
I'd really prefer leaving the ordering of arguments as is: from most significant to least.